.node-version

```diff
@@ -0,0 +1 @@
+14.17.4
```

Comment:
16.x version here

Reply:
Agreed but the next PR. I was accidentally running with node 18 originally and had problems building with it. My nodeenv setup is little creaky at the moment because it got it the way of the sisu sdk publish last week. I will add support for node 16 in the next PR